tests/web3/firefly_getCoverageData0.expected.json

```diff
@@ -25,27 +25,39 @@
         "program": [
           {
             "programCounter": 0,
-            "hitCount": 1
+            "hitCount": 1,
+            "opcode": "PUSH(3)",
+            "argument": "0xffffff"
           },
           {
             "programCounter": 4,
-            "hitCount": 1
+            "hitCount": 1,
+            "opcode": "PUSH(1)",
+            "argument": "0x00"
           },
           {
             "programCounter": 6,
-            "hitCount": 1
+            "hitCount": 1,
+            "opcode": "MSTORE",
+            "argument": null
```

Comment:
`"argument": []`

Reply:
Actually, this should be the `INDEX, VALUE` that `MSTORE` uses off the `<wordStack>`.

Reply:
```
            "programCounter": 6,
            "opcode": "MSTORE",
            "argument": [["0x03", "0x05"], ["0x04", "0x07"]]
```